db/models.py

```diff
@@ -1 +1,42 @@
 from django.db import models
+
+
+class Race(models.Model):
+    RACE_CHOICES = [
+        ("elf", "Elf"),
+        ("dwarf", "Dwarf"),
+        ("human", "Human"),
+        ("ork", "Ork"),
+    ]
+
+    name = models.CharField(max_length=255, unique=True, choices=RACE_CHOICES)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
```

Comment:
The `bonus` field in the `Skill` model is currently a `CharField`. If this field is intended to store numerical values, consider using a `DecimalField` or `IntegerField` instead, depending on the precision required.